src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -864,3 +864,37 @@ You can use this to create context-aware validators.
 ----
 length(title) < 5
 ----
+
+== Pageblock Refresh
```

Comment:
Should we call it page block refresh or page block **auto-**refresh?
Here you use one, lower you use the other.

Pick one and stick with it, preferably the one we use on the UI.

Reply:
We do not have like a grouping or sectional name for it on the UI.  We have an input for autorefresh and a toggle switch for manual refresh. If we had to group both, it would be refresh so I guess *Page block* refresh is okay.
![image](https://user-images.githubusercontent.com/31524469/232465599-85728c44-638e-442f-8b82-eff819e7fafc.png)